marklogic-data-hub-central/ui/src/config/tiles.config.ts

```diff
@@ -82,6 +88,19 @@ const tiles: Record<TileId, TileItem>  = {
         controlColor: '#777',
         controls: ['menu', 'close'],
         intro: 'Monitor Flows and Steps',
+        toolbarHidden: false
+    },
+    detail: {
+        title: 'Explore - Details',
```

Comment:
I think this PR should cover an implementation detail (we're handling the Details view as a separate tile definition) and not how content is presented in the UI. There's no problem with the current titles/organization and changing tile titles (if we wanted to do that) is a bigger question that should involve PM and UX. So let's keep this title as "Explore".